samples/template/README.md

```diff
@@ -1,5 +1,148 @@
 # Savant module template
 
-Start your own project with this template.
+This sample is intended to help users to start developing a custom Savant module. It provides:
+
+* A module with a simple pipeline that includes basic Savant elements and dev features.
+* Scripts with pyfunc and drawfunc templates.
+* A script demonstrating basic Client SDK usage.
```

Comment:
There could be links to the corresponding files (client/run, pyfunc, etc).

Reply:
Added links to files.